apps/sms/views/shim_host.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```

Comment:
> Sorry that I'm not very clear about the shim_host here: Adding the iframe which load this host looks good, but is this host should be shared globally or depends on view?

I was thinking that we should share same iframe file for all view for now. Also in my PR I've just load it in views/inbox/index.html as it's not ready for prime time yet :) But I suspect in your PR you'll be loading it in the main index.html as well.

>  If we treat the host as the same across views, we can move this host and host bootstrap under view/shared folder

Yep, I have no preference over "views/*.html" or "views/shared/*.html" - can move it to shared then :) Host bootstrap file is already there anyway.